src/Abuse/Adapter.php

```diff
@@ -12,4 +12,23 @@ interface Adapter
      * @return bool
      */
     public function check();
+
+    /**
+     * Get all abuse logs
+     *
+     * Returns all the abuse logs that are currently in the DB
+     *
+     * @return array
+     */
+    public function getAllLogs(): array;
```

Comment:
Maybe allow to pass a limit and offset here, and just call getLogs. Might not be a very smart idea to use this method on large DB sets.

Reply:
Done